dashboards/resources/pod.libsonnet

```diff
@@ -59,12 +59,12 @@ local template = grafana.template;
-        'sum by(container) (rate(container_fs_reads_total{%(cadvisorSelector)s, %(diskDeviceSelector)s, %(containerfsSelector)s, %(clusterLabel)s="$cluster", namespace="$namespace", pod="$pod"}[%(grafanaIntervalVar)s]) + rate(container_fs_writes_total{%(cadvisorSelector)s, %(diskDeviceSelector)s, %(containerfsSelector)s, %(clusterLabel)s="$cluster", namespace="$namespace", pod="$pod"}[%(grafanaIntervalVar)s]))' % $._config,
-        'sum by(container) (rate(container_fs_reads_bytes_total{%(cadvisorSelector)s, %(diskDeviceSelector)s, %(containerfsSelector)s, %(clusterLabel)s="$cluster", namespace="$namespace", pod="$pod"}[%(grafanaIntervalVar)s]))' % $._config,
-        'sum by(container) (rate(container_fs_writes_bytes_total{%(cadvisorSelector)s, %(diskDeviceSelector)s, %(containerfsSelector)s, %(clusterLabel)s="$cluster", namespace="$namespace", pod="$pod"}[%(grafanaIntervalVar)s]))' % $._config,
-        'sum by(container) (rate(container_fs_reads_bytes_total{%(cadvisorSelector)s, %(diskDeviceSelector)s, %(containerfsSelector)s, %(clusterLabel)s="$cluster", namespace="$namespace", pod="$pod"}[%(grafanaIntervalVar)s]) + rate(container_fs_writes_bytes_total{%(cadvisorSelector)s, %(diskDeviceSelector)s, %(containerfsSelector)s, %(clusterLabel)s="$cluster", namespace="$namespace", pod="$pod"}[%(grafanaIntervalVar)s]))' % $._config,
+        'sum by(container) (rate(container_fs_reads_total{%(cadvisorSelector)s, %(diskDeviceSelector)s, %(clusterLabel)s="$cluster", namespace="$namespace", pod="$pod"}[%(grafanaIntervalVar)s]))' % $._config,
+        'sum by(container) (rate(container_fs_writes_total{%(cadvisorSelector)s, %(diskDeviceSelector)s, %(clusterLabel)s="$cluster", namespace="$namespace", pod="$pod"}[%(grafanaIntervalVar)s]))' % $._config,
+        'sum by(container) (rate(container_fs_reads_total{%(cadvisorSelector)s, %(diskDeviceSelector)s, %(clusterLabel)s="$cluster", namespace="$namespace", pod="$pod"}[%(grafanaIntervalVar)s]) + rate(container_fs_writes_total{%(cadvisorSelector)s, %(diskDeviceSelector)s, %(clusterLabel)s="$cluster", namespace="$namespace", pod="$pod"}[%(grafanaIntervalVar)s]))' % $._config,
+        'sum by(container) (rate(container_fs_reads_bytes_total{%(cadvisorSelector)s, %(diskDeviceSelector)s, %(clusterLabel)s="$cluster", namespace="$namespace", pod="$pod"}[%(grafanaIntervalVar)s]))' % $._config,
+        'sum by(container) (rate(container_fs_writes_bytes_total{%(cadvisorSelector)s, %(diskDeviceSelector)s, %(clusterLabel)s="$cluster", namespace="$namespace", pod="$pod"}[%(grafanaIntervalVar)s]))' % $._config,
+        'sum by(container) (rate(container_fs_reads_bytes_total{%(cadvisorSelector)s, %(diskDeviceSelector)s, %(clusterLabel)s="$cluster", namespace="$namespace", pod="$pod"}[%(grafanaIntervalVar)s]) + rate(container_fs_writes_bytes_total{%(cadvisorSelector)s, %(diskDeviceSelector)s, %(clusterLabel)s="$cluster", namespace="$namespace", pod="$pod"}[%(grafanaIntervalVar)s]))' % $._config,
```

Comment:
Removing `%(containerfsSelector)s` seems unrelated?

Reply:
Removing `%{containerfsSelector}s` was needed as per this commend [BZ#2074134#c4](https://bugzilla.redhat.com/show_bug.cgi?id=2074134#c4).
The queries like `container_fs_reads_total` or `container_fs_writes_bytes_total` etc don't have `container` field/label in the query result. So any addition of the condition `containerfsSelector` (which is equivalent to `container!=""`) will give you an empty result (`No datapoints found`).


![Screenshot from 2022-11-02 15-51-45](https://user-images.githubusercontent.com/895234/199465443-28206b2a-25ca-4169-ae37-e72bf1945006.png)

Without which we will get all the results,

![Screenshot from 2022-11-02 15-54-05](https://user-images.githubusercontent.com/895234/199465953-4605ffcb-0aed-4d66-9c48-4d89c7153f5e.png)

Reply:
@jan--f , @povilasv ^^^

Reply:
Oh! sorry, the above BZ link may require a Red Hat id. But it mainly talks about the same thing as I mentioned in the above comment

![image](https://user-images.githubusercontent.com/895234/201268258-20cc8f4c-4bdf-45c5-a0fb-8cd595fe2b08.png)

```
Sample query:
container_fs_writes_bytes_total{container!="", cluster="",namespace="namespace-adzg", pod=~"fedorapod-ovijwvtfw-1-5tpt4"}

The query above has a condition which checks `container!=""`, but unfortunately the above query, `container_fs_writes_bytes_total` don't have a result field with named `container`.
Attached a screenshot of query-result fields with the BZ.
```

Extract from the BZ comment